spring-cloud-task-docs/src/main/asciidoc/preface.adoc

```diff
@@ -0,0 +1,44 @@
+[[preface]]
+= Preface
+
+[[task-documentation-about]]
+
+This section provides a brief overview of the Spring Cloud Task reference documentation.
+Think of it as a map for the rest of the document.  You can read this reference guide in a
+linear fashion, or you can skip sections if something doesn't interest you.
+
+== About the documentation
+The Spring Cloud Task reference guide is available as {spring-cloud-task-docs}/html[html],
+{spring-cloud-task-docs}/pdf/spring-cloud-task-reference.pdf[pdf]
+and {spring-cloud-task-docs}/epub/spring-cloud-task-reference.epub[epub] documents. The latest copy
+is available at {spring-cloud-task-docs-current}.
+
+Copies of this document may be made for your own use and for
+distribution to others, provided that you do not charge any fee for such copies and
+further provided that each copy contains this Copyright Notice, whether distributed in
+print or electronically.
+
+[[task-documentation-getting-help]]
+== Getting help
+Having trouble with Spring Cloud Task, We'd like to help!
+
+* Ask a question - we monitor http://stackoverflow.com[stackoverflow.com] for questions
+  tagged with http://stackoverflow.com/tags/spring-cloud[`spring-cloud`].
+* Report bugs with Spring Cloud Task at https://github.com/spring-cloud/spring-cloud-task/issues.
+
+NOTE: All of Spring Cloud Task is open source, including the documentation! If you find problems
+with the docs; or if you just want to improve them, please {github-code}[get involved].
```

Comment:
This is the same syntax that Boot is using.  I think it needs to be deployed on github to work...